docs/dev/new_check_howto.md

```diff
@@ -441,7 +441,10 @@ Our example integration has a very simple `awesome/manifest.json`, the bulk of w
   "is_public": false,
   "integration_id": "awesome",
   "assets": {
-    "dashboards": {},
+    "dashboards": {
+      "Awesome Overview": "assets/dashboards/overview.json",
```

Comment:
I think this example is also used for integrations extras that do not have dashboards

Reply:
Yes, I'm not sure if we should add it here either.

This document seems half way a reference documentation and a tutorial.

Reply:
yup, and that's a known issue for the doc team. We have started refactoring the developer section of the doc, but the first step was to focus on data collected doc: https://github.com/DataDog/documentation/pull/5281 

This documentation should be split into a Guide/ Getting Started and a pure reference doc in the mid-term

Reply:
I think we can merge it for now, and we will update it later one.